rasa/core/actions/action.py

```diff
@@ -168,6 +176,56 @@ def __str__(self) -> Text:
         return "Action('{}')".format(self.name())
 
 
+class ActionUtterPredictedResponse(Action):
```

Comment:
do we want to call it `ActionRetrieveResponse` or something?

Reply:
Fair enough! Changing